cumulus/parachains/integration-tests/emulated/tests/bridges/bridge-hub-westend/Cargo.toml

```diff
@@ -41,6 +42,7 @@ bridge-hub-westend-runtime = { workspace = true }
 cumulus-pallet-xcmp-queue = { workspace = true }
 emulated-integration-tests-common = { workspace = true }
 parachains-common = { workspace = true, default-features = true }
+penpal-emulated-chain = { workspace = true }
 rococo-westend-system-emulated-network = { workspace = true }
```

Comment:
you don't need to take direct dependency to `penpal-emulated-chain` here, it is already included in `rococo-westend-system-emulated-network`

Reply:
Removed in https://github.com/vgeddes/polkadot-sdk/blob/snowbridge-v2/cumulus/parachains/integration-tests/emulated/tests/bridges/bridge-hub-westend/Cargo.toml.